### PR TITLE
Enable onboarding wizard for everybody and remove the feature flag

### PR DESCRIPTION
**Pull request by mporkola**
```bash
~/p/sharetribe git:enable-onboarding-wizard ❯❯❯ git grep "onboarding_redesign_v1" | wc -l
       0
```

### CODE REVIEW

File: app/controllers/admin/communities_controller.rb [hunk 1]
```diff
@@ -9,11 +9,7 @@ def getting_started
     @selected_left_navi_link = "getting_started"
     @community = @current_community
 
-    if(feature_enabled?(:onboarding_redesign_v1))
-      redirect_to admin_getting_started_guide_path
-    else
-      render locals: {paypal_enabled: PaypalHelper.paypal_active?(@current_community.id)}
-    end
+    redirect_to admin_getting_started_guide_path
```
Comment (Gnito) [inline app/controllers/admin/communities_controller.rb]: This should be done in the left navigation bar

Comment (rap1ds) [inline app/controllers/admin/communities_controller.rb]: We could now remove this whole action and also the related `getting_started.haml` (and all the related translations)

Comment (rap1ds) [inline app/controllers/admin/communities_controller.rb]: That can be also done in a separate PR if we just want to quickly enable onboarding for everyone, but anyhow it's good to do it.
